lib/workload/stateless/filemanager/filemanager/src/env.rs

```diff
@@ -1,10 +1,10 @@
 //! Handle environment variables.
 //!
 
-use dotenvy::dotenv;
 use std::ffi::OsStr;
```

Comment:
Interesting, why is ffi::OsStr needed here?

Reply:
It doesn't have to be, I was just copying the `env::var` types: https://doc.rust-lang.org/std/env/fn.var.html, although it could just as well be `K: AsRef<str>`